src/games/stendhal/server/core/engine/StendhalRPRuleProcessor.java

```diff
@@ -189,6 +190,9 @@ public void setContext(final RPServerManager rpman) {
 				logger.info("No server extensions configured in ini file.");
 			}
 
+			// conditionally set freeze hell schedule
+			FreezeHell.onStartup();
```

Comment:
Let's use the normal way of adding quests or configure zones instead of special code in StendhalRPRuleProcessor

Reply:
That makes a lot more sense. I would feel better had I thought of it myself. ;)

Reply:
Changed to use ZoneConfigurator with `-Dstendhal.nohellfreezer` to disable.